### ISSUE TITLE
[Nagrand] Quest: Body of Evidence & Returning the Favor

### ISSUE THREAD

**Issue by ATalisan** [Valhalla-Gaming/Tracker#5427]
After completing Quest: Ruthless Cunning given by Lantresor of the Blade in Nagrand he gives no further quests, though he should give Body of Evidence and Returning the Favor

Link: Ruthless Cunning - http://wowwiki.wikia.com/wiki/Quest:Ruthless_Cunning

**Comment by Tregeta** [comment]
The prerequisites for Body of Evidence quest are Armaments for Deception and Ruthless Cunning.

To be able to get this quest from him as Horde you need to do the quest chain: //www.wowhead.com/?quest=9888

And for Alliance the quest chain: //www.wowhead.com/?quest=9917

**Comment by ATalisan** [comment]
Yes, on the Alliance side, I did complete all of those prerequisites listed in your link as a 69-70 Night Elf Druid.
